# Livox publish pointcloud msg format.

Header header
uint64 timebase
uint32 timestep
uint32 point_num
uint8  lidar_id
uint8[3]  rsvd
CustomPoint[] points

